master/internal/command/command.go

```diff
@@ -195,6 +297,10 @@ func (c *command) Receive(ctx *actor.Context) error {
 			JobActor: ctx.Self(),
 		})
 
+		ctx.Ask(c.allocation, actor.Ping{}).Get()
```

Comment:
this is a bit of a risk - if it gets resources immediately, it will also ask us back to build the task spec..

Reply:
it's the same as trial restoration. if I were asking for `getSummary` like `createGenericActor` does, then yeah it hangs because of this, but works okay for Ping.